molecule/delegated/tests/motd/debian.py

```diff
@@ -14,3 +14,5 @@ def test_pkg(host):
     check_ansible_os_family(host)
 
     assert not host.package("update-motd").is_installed
+
+    # There isn't a direct equivalent to the "update-motd" package on CentOS.
```

Comment:
Is this really needed? There is no redhat.py which should be conclusive enough. Adding a comment here about centos seems out of place (but i know why you put it here of course)

Reply:
deleted